solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx

```diff
@@ -2,22 +2,42 @@ import React from "react"
 import { withBaseModal } from "../withBaseModal"
 import { ModalBody, ModalFooter, ModalHeader } from "../Modal"
 import Button from "../../Button"
+import { colors } from "../../../constants/colors"
+import { GRANT_MANAGER_EMAIL } from "../../../constants/constants"
+
+const styles = {
+  header: {
+    backgroundColor: colors.yellow30,
+    color: colors.yellowSecondary,
+    borderBottom: `1px solid ${colors.yellowSecondary}`,
+  },
+}
 
 export const ContactYourGrantManagerWarning = withBaseModal(
   ({
     header = "Contact your grant manager",
-    bodyTitle = "To enable staking into Threshold please contact your grant manager first.",
+    bodyTitle = "To enable your stake in Threshold you have to contact your Grant Manager",
+
     onClose,
   }) => {
     return (
       <>
-        <ModalHeader>{header}</ModalHeader>
+        <ModalHeader style={styles.header}>{header}</ModalHeader>
         <ModalBody>
           <h3>{bodyTitle}</h3>
+          <p className="text-grerey-70 mt-2">
+            To use this stake in Threshold, you will have to contact your grant
+            manager&nbsp;
+            <a href={`mailto:${GRANT_MANAGER_EMAIL}`}>
+              ({GRANT_MANAGER_EMAIL})
+            </a>
+            .
+          </p>
+          <p className="mt-2">Please do it immediately!</p>
         </ModalBody>
         <ModalFooter>
-          <Button className={"btn btn-secondary btn-lg"} onClick={onClose}>
-            close
+          <Button className="btn btn-unstyled text-link" onClick={onClose}>
+            Close
```

Comment:
Should be

```suggestion
            Cancel
```

as it is in Figma designs

Reply:
[244a15e](https://github.com/keep-network/keep-core/pull/2853/commits/244a15ea0b9c2192fc473b26ba4b2e3520724f3a)